lib/models/LabelReallocationRequest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ *  Copyright 2015 SmartBear Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ *
+ *
+ * NOTE: This class is auto generated by the swagger code generator program. Do not edit the class manually.
+ *
+ */
+
+namespace DwollaSwagger\models;
+
+use \ArrayAccess;
+
+class LabelReallocationRequest implements ArrayAccess {
+  static $swaggerTypes = array(
+      'from_label_id' => 'string',
```

Comment:
Should these two ID attributes be replaced with `_links`? https://docs.dwolla.com/#create-a-label-reallocation

Reply:
Ah, good catch. I'll make that change quick